text/bayes.go

```diff
@@ -160,6 +161,30 @@ type NaiveBayes struct {
 	Output io.Writer
 }
 
+// histogram allows conncurrency-friendly map access via its
+// exported Get and Set methods
+type histogram struct {
+	sync.RWMutex
+	words map[string]Word
+}
+
+// Get looks up a word from h's Word map, it should be used in
+// place of a direct map lookup
```

Comment:
can we even out the comment length on these a bit? eg.

```go
// Get looks up a word from h's Word map, it should be used in
// place of a direct map lookup the only caveat here is that it will
// always return the 'success' boolean.
```